mu4/notation/view/notationtoolbarmodel.h

```diff
@@ -48,23 +53,35 @@ class NotationToolBarModel : public QAbstractListModel, public async::Asyncable
     Q_INVOKABLE void load();
     Q_INVOKABLE void click(const QString& action);
 
+    Q_INVOKABLE QVariantMap get(int index);
```

Comment:
Please, rename this method to something like noteInputItem

Reply:
The GridViewSectional component calls this method on the model. The method is named so that both C ++ and qml models can be used.